iscservice/ut/service_ut.c

```diff
@@ -885,7 +900,8 @@ static void test_remote_waiting(void)
 
 	/* Wait for reply fop. */
 	m0_chan_wait(&waiter);
-	phase = remote_invocation_async(0, BT_INLINE, phase, &arg_fop);
+	phase = remote_invocation_async(&fid, 0, BT_INLINE, phase, &arg_fop);
+	m0_fop_put0_lock(arg_fop);
```

Comment:
Is this already called inside the remote_invocation_async()?

Reply:
m0_fop_put0_lock() is there inside remote_invocation_async(), 
but it does not call m0_fop_fini() in it, because ref->ref_cnt is not 0 at that point.
Hence called again.